### DISCUSSION TITLE
API - Unique Identifier

### DISCUSSION THREAD

**Discussion by Angeschossen** [WeblateOrg/weblate#13318] [General] [answered]
Is see there is an "id" property when calling /api/user/username/
Is this id unique and always stays the same? Also, is there a way to retrieve information about a user with the id?

If not, can we somehow prevent users from changing their username?

Thank you.

**Comment by nijel** [comment, marked as answer]
Yes, id is fixed. Since 0ff4ed299126cbe749febc4c2d703dfa000e3585 you will be able to search by id (`/api/users/?id=N`).

**Reply by Angeschossen** [reply]
Thanks a lot! This will be very helpful. :)
